.ci/scripts/test-GC-manifest.sh

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env bash
+set -eo pipefail
```

Comment:
It would be nice if you describe how to run this test locally in https://github.com/kyma-incubator/marketplaces/blob/master/google-cloud/README.md